### DISCUSSION TITLE
Is there a fix for Cloudflare (cclonline.com) for example?

### DISCUSSION THREAD

**Discussion by benjaminfphillips** [jef/streetmerchant#1704] [Help]
Hello all,

Want to start by saying I love this tool and appreciate jef for sharing with everyone else.

My question was surrounding the Cloudflare DDOS protection "wait at least 5 seconds" message that I see every single time I open the cclsite using StreetMerchant? Is there a way around this? Looking at the issues jef pushed some kind of code fix mid December but I only downloaded this today so would expect it to already be in the code but I still end up in an endless loop when trying to access any of the pages for the 30 series cards.

Any input would be greatly appreciated.

Thanks,

**Comment by 85256638** [comment]
I'm getting the same issue

**Comment by Karel1415** [comment]
StreetMerchant still doesn't work on cclonline regardless the settings. But I can open the page in any of my browsers manually.
